Sources/Utils/Log.cpp

```diff
@@ -31,6 +40,12 @@ FILE *sOutputStream = stdout;
 #else
 FILE *sOutputStream = stderr;
 #endif
+
+// PATH_MAX isn't defined when targeting MSVC
+#ifndef PATH_MAX
+#define PATH_MAX 1024
```

Comment:
Where is 1024 coming from? I'm assuming this is only for Windows platforms? Consider using `MAX_PATH` instead if that's the case.